src/js/scrollArea.jsx

```diff
@@ -75,12 +79,12 @@ export default class ScrollArea extends React.Component{
             warnAboutElementChild();
         }
 
-        var classes = 'scrollarea ' + className;
-        var contentClasses = 'scrollarea-content ' + contentClassName
+        var classes = 'scrollarea ' + (className || '');
```

Comment:
Without this, if the className is not passed the resulting html will be: `class="scrollarea-content undefined"`